script/utils/constants.json

```diff
@@ -1,181 +1,245 @@
+    "OPTIMISM_SEPOLIA": 11155420,
+    "POLYGON_AMOY": 80002,
+    "SCROLL_SEPOLIA": 534351,
+    "SEPOLIA": 11155111,
+    "UNICHAIN_SEPOLIA": 1301,
+    "ZK_SYNC_SEPOLIA": 300,
+    "SOLANA_DEVNET": 133268194659241
   },
   "oftEids": {
-    "ARBITRUM": 30110,
-    "ARBITRUM_SEPOLIA": 40231,
-    "BASE": 30184,
-    "BASE_SEPOLIA": 40245,
-    "BLAST": 30243,
-    "BSC": 30102,
     "MAINNET": 30101,
     "OPTIMISM": 30111,
-    "OPTIMISM_SEPOLIA": 40232,
+    "BSC": 30102,
+    "UNICHAIN": 30320,
     "POLYGON": 30109,
+    "LENS": -1,
+    "BOBA": -1,
+    "ZK_SYNC_SEPOLIA": -1,
+    "ZK_SYNC": -1,
+    "WORLD_CHAIN": 30319,
+    "REDSTONE": -1,
+    "MODE_SEPOLIA": -1,
+    "LISK": -1,
+    "UNICHAIN_SEPOLIA": 40333,
+    "SONEIUM": 30340,
+    "LISK_SEPOLIA": -1,
+    "SUPERSEED": -1,
+    "BASE": 30184,
+    "MODE": -1,
+    "LENS_SEPOLIA": -1,
+    "ALEPH_ZERO": -1,
+    "ARBITRUM": 30110,
+    "INK": -1,
+    "LINEA": -1,
     "POLYGON_AMOY": 40267,
+    "BLAST": 30243,
+    "BASE_SEPOLIA": 40245,
+    "TATARA": -1,
+    "ARBITRUM_SEPOLIA": 40231,
+    "SCROLL_SEPOLIA": -1,
+    "SCROLL": -1,
+    "BOB_SEPOLIA": -1,
+    "ZORA": -1,
     "SEPOLIA": 40161,
+    "OPTIMISM_SEPOLIA": 40232,
+    "BLAST_SEPOLIA": -1,
     "SOLANA": 30168,
-    "SOLANA_DEVNET": 40168,
-    "SONEIUM": 30340,
-    "UNICHAIN": 30320,
-    "UNICHAIN_SEPOLIA": 40333,
-    "WORLD_CHAIN": 30319
+    "SOLANA_DEVNET": 40168
   },
   "wrappedNativeTokens": {
     "MAINNET": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-    "SEPOLIA": "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9",
-    "ARBITRUM": "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1",
-    "ARBITRUM_SEPOLIA": "0x980B62Da83eFf3D4576C647993b0c1D7faf17c73",
-    "BSC": "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c",
-    "POLYGON": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
-    "POLYGON_AMOY": "0xA5733b3A8e62A8faF43b0376d5fAF46E89B3033E",
-    "ZK_SYNC": "0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91",
     "OPTIMISM": "0x4200000000000000000000000000000000000006",
-    "OPTIMISM_SEPOLIA": "0x74A4A85C611679B73F402B36c0F84A7D2CcdFDa3",
-    "BASE": "0x4200000000000000000000000000000000000006",
-    "BASE_SEPOLIA": "0x999B45BB215209e567FaF486515af43b8353e393",
-    "LENS": "0x6bDc36E20D267Ff0dd6097799f82e78907105e2F",
-    "LINEA": "0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f",
-    "LINEA_SEPOLIA": "0x10253594A832f967994b44f33411940533302ACb",
-    "SCROLL_SEPOLIA": "0x5300000000000000000000000000000000000004",
-    "SCROLL": "0x5300000000000000000000000000000000000004",
+    "BSC": "0x2170Ed0880ac9A755fd29B2688956BD959F933F8",
     "UNICHAIN": "0x4200000000000000000000000000000000000006",
-    "UNICHAIN_SEPOLIA": "0x4200000000000000000000000000000000000006",
-    "ALEPH_ZERO": "0xB3f0eE446723f4258862D949B4c9688e7e7d35d3",
-    "BLAST": "0x4300000000000000000000000000000000000004",
-    "BLAST_SEPOLIA": "0x4200000000000000000000000000000000000023",
+    "POLYGON": "0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619",
```

Comment:
We have different addresses for Polygon here - the new address is incorrect because it's WETH.